x-pack/tslint.yaml

```diff
@@ -30,3 +30,7 @@ rules:
        * specific language governing permissions and limitations
        * under the License.
        */
+  module-migration:
```

Comment:
Pretty sure this overrides the module-migration config in the root, can we just place this there?